sdv/tabular/base.py

```diff
@@ -471,6 +472,8 @@ def sample(self, num_rows=None, max_retries=100, max_rows_multiplier=10,
             else:
                 transformed_conditions_in_group = transformed_conditions.loc[condition_indices]
                 transformed_groups = transformed_conditions_in_group.groupby(transformed_columns)
+                if len(transformed_groups.groups) == 0:
+                    raise InvalidConditionsError
```

Comment:
Add a break line for consistency

Reply:
I'm not sure about defining a new type of constraint for this. I would rather just use a `ValueError` and put the proper message on it, since this is just an invalid user input scenario.

Reply:
Follow up question - what is the disadvantage to defining a custom error type that extends a standard one (e.g. `ValueError`). Is it confusing for users?